lib/widget/evaluation_form/post-op-home/surgical_incision_form.dart

```diff
@@ -278,17 +278,17 @@ class AdvisePage extends StatelessWidget {
                         padding: const EdgeInsets.all(5.0),
                         child: Column(
                           children: [
-                            Text(
-                                'การดูแลเพื่อส่งเสริมการหายของแผลและป้องกันการติดเชื้อที่แผลผ่าตัด มีดังนี้'),
+                            Text('''
+การดูแลเพื่อส่งเสริมการหายของแผลและป้องกันการติดเชื้อที่แผลผ่าตัด มีดังนี้'''),
```

Comment:
Why we need tripple single qoute here ?

Reply:
Fixed an problem where vscode warned that Avoid line longer than 80 characters.